src/Effect/Exec.hs

```diff
@@ -145,7 +146,15 @@ instance ToDiagnostic ExecErr where
               , "stderr: " <> line <> indent 2 (pretty @Text (decodeUtf8 (cmdFailureStderr err)))
               ]
           )
-    CommandParseError cmd err -> "Failed to parse command output. command: " <> viaShow cmd <> " . error: " <> pretty err
+    CommandParseError cmd err ->
+      vsep
+        [ "Failed to parse command output. command: " <> viaShow cmd <> "."
+        , ""
+        , "Details:"
```

Comment:
[nit] upon testing locally, I think killing this "details" line improves the formatting a little bit